aiida_common_workflows/cli/launch.py

```diff
@@ -90,6 +91,10 @@ def cmd_relax(  # pylint: disable=too-many-branches
 
         return
 
+    if not isinstance(engine_options, dict):
+        message = f'You must pass a dictionary in JSON format (it is now {type(engine_options)}'
+        raise click.BadParameter(message, param_hint='engine-options')
```

Comment:
Don't have to type check here do we since this is done by the parameter?

Reply:
Not really - e.g. the string `1` is a valid JSON, and is of type `int` and not `dict` (I also at the beginning had assumed it was always a dict :-) )